docs/20_tutorial.md

```diff
@@ -14,10 +15,161 @@ the computers read those ballots, to determine whether the computers
+for chain of custody logs. Nor does it provide any guidance for ballot 
+interpretation. 
+
+## For Secretary of State Users
+
+### Logging In
+
+Screenshot (SoS authentication) ---
+
+### Entering the Risk Limit
+
+To enter the Risk Limit for comparison audits:`
+Screenshot (SoS dash?) --- 
+
+### Entering the Random Seed
+
+To enter the random seed:
+Screenshot (SoS dash?) ---
+
+### Selecting Contests
+
+To select the contests that will drive the sample size and stop/go
+decisions for each round:
+Screenshot (SoS dash?) ---
+
+### Reading the Number of Ballots to Audit
+
+The RLA Tool calculates the number of ballots to audit in each round of the RLA. To read
+the number of ballots for each round:
+Screenshot (SoS dash?) ---
+
+### Declaring a Full Hand Count
+
+Screenshot ---
+
+
+Basic overview: Screenshots of the dashboard/tool, ordered in the sequence
+they will be encountered, with brief, clear instructions describing what
+is expected of the user(s) at each step, starting with the login screen
+and proceeding through the completion of the audit, or determination that
+a full hand count is required. First SOS, then County, then Public.
+
+## For County Administrators
+
+For the 2017 Colorado implementation, the full functionality of the RLA Tool requires exports from the 
+Dominion System ???. 
+***Colorado2017
+***which Dominion system?
+
+### Logging In
+
+Screenshot (County Admin Authentication) ---
+
+
+### <a name="comparison-audit-upload"></a>Comparison Audit Upload
+
+To prepare for upload the 
+ballot manifest and the CVR file must be first verified and hashed. 
+The RLA Tool does not provide a utility for the 
+verification and hashing. 
+***ballot manifest upload prep
+
+Once the ballot manifest and CVR files are verified and hashed, 
+they can be uploaded.
+
+Screenshot (page:county) ---
+
+Note that the RLA Tool does not support export of the
+tabulation results to the Election Night Reporting System. 
+
+### Ballot Polling Audit Upload
+The RLA Tool is designed for comparison audits, which require a voting
+system capable of exporting CVR files. However, Counties whose voting systems do not
+export CVR files can use the RLA Tool to upload ballot manifests, described
+in the [Comparison Audit Upload](#comparison-audit-upload) 
+section.
+
+### List of Ballot for Audit
+To see the list of ballots to be audited in the next round:
+Screenshot (???) ---
+
+
+
+## For County Audit Boards
+========
+
+### Signing In
+
+The Audit Board does not log directly into the RLA Tool with
+computer system usernames and passwords. However, whenever 
+the Audit Board begins to interact with the RLA Tool, either 
+at the beginning of an audit round or after taking a break, 
+the Audit Board must sign in.
+
+Screenshot (page:Acme County) ---
+
+Click on "Start Audit"
+
+Screenshot (page:Audit Board Sign In) ---
+
+Click "Next"
+
+### Auditing Ballots
+
+Note: do *not* use your browser's "back" arrow during the audit process.
```

Comment:
Just saw this. I believe this is an overstatement: the worst the back button would do right now is take them to the wrong page, if they hit it during the audit wizard. If they navigate back to the correct URL path, they will see the correct thing. The browser back button works generally. We will change the wizard to be fully browser-back friendly, though.